### ISSUE TITLE
[SF][A11y][Switch | Select]: <svg> elements in Switch and Select components missing accessible text

### ISSUE THREAD

**Issue by sleric1024** [SAP/ui5-webcomponents-react#7137]
### Describe the bug

Description
When using the UI5 Switch and Select components, aXe scans and our internal accessibility testing team both flag an issue related to <svg> elements. Specifically:

Ensure <svg> elements with an img, graphics-document or graphics-symbol role have an accessible text.

This indicates that the SVG icons in these components do not have a proper accessible name, causing them to fail accessibility guidelines. As a result, screen readers are not able to convey the purpose or meaning of these icons to visually impaired users.

### Isolated Example

https://aoniryymmmgithub-2lg2--5173--5a421e5b.local-credentialless.webcontainer.io/

### Reproduction steps

1.  Render a UI5 Switch or Select  from https://aoniryymmmgithub-2lg2--5173--5a421e5b.local-credentialless.webcontainer.io/
2. Run an AXE accessibility scan (or any other standard accessibility scanner) on the page.
3. Observe the reported violation concerning the SVG element lacking an accessible text.

### Screenshots or Videos

![Image](https://github.com/user-attachments/assets/69add96e-3b18-463a-8653-0d3691f682ae)

### UI5 Web Components for React Version

2.8.1

### UI5 Web Components Version

2.8.0

### Browser

Chrome

### Relevant log output

```shell

```

### Organization

SuccessFactors

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @sleric1024 

I will close this issue as there is already another one covering this topic. If you have any questions or more information, please address them in the [original issue](https://github.com/SAP/ui5-webcomponents/issues/10986). The linked issue was only created for the `Select` component, but the linked PR fixes it for every component using an `ui5-icon`.
